be/src/storage/utils.cpp

```diff
@@ -390,4 +391,14 @@ std::string file_name(const std::string& fullpath) {
     return path.filename().string();
 }
 
+std::string _mask_token(const std::string& str) {
```

Comment:
I don't think this is good enough. the reason is, you can't predicate the name of the query parameters whose values should be treated as sensitive.

Also, for a http url, e.g. http://user:<password>@host:port/path/segment/... , the `<password>` part is also taken as sensitive info, although it is not suggested to do it like this for http plain text auth.